orcas/l1l2.go

```diff
@@ -615,7 +636,33 @@ func (l *L1L2Orca) Get(req common.GetRequest) error {
 
 					if err != nil {
 						metrics.IncCounter(MetricCmdGetSetErrorsL1)
-						return err
+
```

Comment:
The change in Get may not be necessary since a Set failure has this consequences, which seem acceptable: (1) no record in memcached, (2) new record actually there, (3) some other thread concurrently had set the same key.